checks.d/mysql.py

```diff
@@ -32,78 +33,341 @@
     'Com_insert': ('mysql.performance.com_insert', RATE),
     'Com_update': ('mysql.performance.com_update', RATE),
     'Com_delete': ('mysql.performance.com_delete', RATE),
+    'Com_replace': ('mysql.performance.com_replace', RATE),
+    'Com_load': ('mysql.performance.com_load', RATE),
     'Com_insert_select': ('mysql.performance.com_insert_select', RATE),
     'Com_update_multi': ('mysql.performance.com_update_multi', RATE),
     'Com_delete_multi': ('mysql.performance.com_delete_multi', RATE),
     'Com_replace_select': ('mysql.performance.com_replace_select', RATE),
+    # Connection Metrics
+    'Connections': ('mysql.net.connections', RATE),
+    'Max_used_connections': ('mysql.net.max_connections', GAUGE),
+    'Aborted_clients': ('mysql.net.aborted_clients', RATE),
+    'Aborted_connects': ('mysql.net.aborted_connects', RATE),
+    # Table Cache Metrics
+    'Open_files': ('mysql.performance.open_files', GAUGE),
+    'Open_tables': ('mysql.performance.open_tables', GAUGE),
+    # Network Metrics
+    'Bytes_sent': ('mysql.performance.bytes_sent', RATE),
+    'Bytes_received': ('mysql.performance.bytes_received', RATE),
+    # Query Cache Metrics
     'Qcache_hits': ('mysql.performance.qcache_hits', RATE),
-    'Innodb_mutex_spin_waits': ('mysql.innodb.mutex_spin_waits', RATE),
-    'Innodb_mutex_spin_rounds': ('mysql.innodb.mutex_spin_rounds', RATE),
-    'Innodb_mutex_os_waits': ('mysql.innodb.mutex_os_waits', RATE),
+    'Qcache_inserts': ('mysql.performance.qcache_inserts', RATE),
+    'Qcache_lowmem_prunes': ('mysql.performance.qcache_lowmem_prunes', RATE),
+    # Table Lock Metrics
+    'Table_locks_waited': ('mysql.performance.table_locks_waited', GAUGE),
+    # Temporary Table Metrics
     'Created_tmp_tables': ('mysql.performance.created_tmp_tables', RATE),
     'Created_tmp_disk_tables': ('mysql.performance.created_tmp_disk_tables', RATE),
     'Created_tmp_files': ('mysql.performance.created_tmp_files', RATE),
+    # Thread Metrics
+    'Threads_connected': ('mysql.performance.threads_connected', GAUGE),
+    'Threads_running': ('mysql.performance.threads_running', GAUGE),
+    # MyISAM Metrics
+    'Key_buffer_bytes_unflushed': ('mysql.myisam.key_buffer_bytes_unflushed', RATE),
+    'Key_buffer_bytes_used': ('mysql.myisam.key_buffer_bytes_used', GAUGE),
```

Comment:
Ok so I don't think `Key_buffer_bytes_unflushed` is a `rate` either. I think we should change that to a `gauge`. Reading the docs about the base metric `Key_blocks_not_flushed`:  "The number of key blocks in the MyISAM key cache that have changed but have not yet been flushed to disk." -- Because flushing will occur, this will _not_ behave as a rate I believe.